### ISSUE TITLE
Talos II - 0.6 Release - Cannot boot if no TPM - TPM module cannot be expected (cannot be bought)

### ISSUE THREAD

**Issue by tlaurion** [Dasharo/dasharo-issues#191]
USB boot results in:
```
TPM_OpenClientCharDev: Could not open char device /dev/tpm0: No such file or directory
Error I/O error from TPM_Extend
!!! Could not find any ISO, trying bootable USB
TPM_OpenClientCharDev: Could not open char device /dev/tpm0: No such file or directory
Error I/O error from TPM_Extend
Failed to extend PCR 4
!!!!! Something failed during USB boot
TPM_OpenClientCharDev: Could not open char device /dev/tpm0: No such file or directory
Error I/O error from TPM_Extend
!!!!! Starting recovery shell
```

Internally, this requires TPM to extend PCR4 with kernel modules hashes prior of loading them.

```
~ # usb-scan
!!! Could not find any ISO, trying bootable USB
TPM_OpenClientCharDev: Could not open char device /dev/tpm0: No such file or directory
Error I/O error from TPM_Extend
Failed to extend PCR 4
```

I think 0.6.1 needs to be released prior of going further, which should be based on https://github.com/osresearch/heads/commit/493149b8d2ab1046537f240919038364669dcc46

Otherwise, willing testers will become unwilling testers. 
We cannot consider any tester today to have non-buyable assembled TPM module.

**Comment by macpijan** [comment]
See [here](https://github.com/Dasharo/dasharo-issues/issues/192#issuecomment-1259249658) for TPM availability.

**Comment by miczyg1** [comment]
We will need a v0.7.0 release or something to close it.

**Comment by macpijan** [comment]
TPM module can be bought here: https://shop.3mdeb.com/shop/modules/tpm-1-2-compatible-with-raptor-computing-systems-talos-ii-mainboard/

The design files are here: https://github.com/3mdeb/talos-tpm-module

What is the expectation here @tlaurion ? Do we want heads on Talos to boot without TPM? What is heads usefulness in such a case? Are there more platforms like this? Do we need to test for the "booting heads without TPM" for each release?

Or maybe it can be closed, as TPM is already available, as this was the main point in the original comment.

**Comment by macpijan** [comment]
According to: https://github.com/Dasharo/dasharo/pull/43#discussion_r1273470307 we should be able to close this one.

**Comment by SergiiDmytruk** [comment]
Some context: this was opened after https://github.com/osresearch/heads/pull/1200 was already merged which happened after v0.6.0 release.  The release of v0.6.1 was suggested to pick up that change and avoid requiring TPM presence.  So, yes, this should be fixed.
